### PR TITLE
Getters

### PR DESCRIPTION
**Pull request by TamaraRuzic**

### CODE REVIEW

File: board.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        "enterBoardName" : "input[placeholder='Enter title...']",
+        "prevButton" : "button[name='prev_btn']",
+        "nextButton" : "button[name='next_btn']",
+        "scrumType" : "span[name='type_scrum']",
+        "kanbanType" : "span[name='type_kanban']",
+        "uploadLogo" : "svg[slot='dropdown-button']",
+        "closeModalBtn" : "button[class='el-button vs-c-close-modal el-button--text el-button--small']"
+    },
+    "boardSidebar" : {
+        "productBacklog" : "a[class='vs-c-site-logo router-link-exact-active router-link-active']",
```
Comment (bdimitrovski) [inline board.json:15]: What if the project ID was different?

File: login.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "loginEmail" : "input[data-vv-as='email']",
```
Comment (bdimitrovski) [inline login.json:2]: data-vv-as might not be the best possible selector in this case.

File: login.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    "forgotPass" : "a[class='vs-c-auth-modal__body-text vs-c-auth-modal__body-text--small']",
+    "submitBtn" : "button[type='submit']",
+    "backHome" : "a[class='vs-u-font-bold vs-u-text--uppercase vs-c-auth-modal__body-text']",
+    "googleSignUp" : "button[class='vs-c-btn vs-c-btn--social-auth vs-c-btn--gp disabled']",
+    "fbSignUp" : "button[class='vs-c-btn vs-c-btn--social-auth vs-c-btn--fb']",
+    "tweeterSignUp" : "button[class='vs-c-btn vs-c-btn--social-auth vs-c-btn--tw']",
+    "regzenSignUp" : "button[class='vs-c-btn vs-c-btn--social-auth vs-c-btn--regzen']",
+    "alreadyHaveAcc" : "a[class='vs-c-btn vs-c-btn--link']",
+    "newAccount" : "a[class='vs-u-font-bold vs-c-auth-modal__bottom-text-main vs-u-text--uppercase']"
+}
```
Comment (bdimitrovski) [inline login.json:13]: Newline

File: board.json (hunk 1 above)
Comment (TamaraRuzic) [reply board.json:15]: I would say it is better to go with `div[class="vs-l-project__menu"] > u[class="vs-c-list"] > li:nth-child()` selector.

File: register.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
+    "backToPricing" : "span[class='vs-u-text--uppercase vs-c-auth-modal__body-text vs-u-font-lg vs-u-gap--left-sm']",
+    "registerEmail" : "form[class='el-form'] > div:nth-child(1) > div[class='el-form-item__content'] > div[class='el-input'] > input[type='email']",
+    "registerPass" : "form[class='el-form'] > div:nth-child(2) > div[class='el-form-item__content'] > div[class='el-input'] > input[type='password']",
+    "showPass" : "button[class='vs-c-btn--icon vs-u-color--modal-body-text']",
+    "numberOfUsers" : "input[data-vv-as='number of users']",
+    "acceptTerms" : "span[class='vs-c-checkbox-check]",
+    "pricePlanMonth" : "div[class='vs-c-switch-pricing-plan-list']:nth-child(0)",
+    "pricePlanAnnaul" : "div[class='vs-c-switch-pricing-plan-list']:nth-child(1)",
```
Comment (bdimitrovski) [inline register.json:9]: Slight typo here (`pricePlanAnnaul`), make sure to watch out for these :)

**Review by bdimitrovski — APPROVED** [review]
Nice!

**Pull request merged**